src/config/lang/english.go

```diff
@@ -276,7 +276,7 @@ $ zarf package mirror-resources <your-package.tar.zst> \
 	CmdPackageDeployFlagShasum                         = "Shasum of the package to deploy. Required if deploying a remote package and \"--insecure\" is not provided"
 	CmdPackageDeployFlagSget                           = "[Deprecated] Path to public sget key file for remote packages signed via cosign. This flag will be removed in v1.0.0 please use the --key flag instead."
 	CmdPackageDeployFlagSkipWebhooks                   = "[alpha] Skip waiting for external webhooks to execute as each package component is deployed"
-	CmdPackageDeployFlagTimeout                        = "Timeout for Helm operations such as installs and rollbacks"
+	CmdPackageDeployFlagTimeout                        = "Timeout for health checks and Helm operations such as installs and rollbacks"
```

Comment:
Do we want the existing `--timeout` flag to apply to health checks or should we implement a new flag. Alternatively, we could add timeout to the schema, though I'm not a big fan of that, especially if we want to add a wait flag that works with /health checks as well.